stock_analytic/models/stock.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# © 2013 Julius Network Solutions
+# © 2015 Clear Corp
+# © 2016 Andhitia Rama <andhitia.r@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models, api
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    account_analytic_id = fields.Many2one(
```

Comment:
IMHO i think that the name of the field should be analytic_account_id. I think the name of the field should match the name of other modules as for example the account module https://github.com/OCA/OCB/blob/8.0/addons/account/account.py#L2398 or the project module https://github.com/OCA/OCB/blob/8.0/addons/project/project.py#L231